persistence/src/commonMain/db_user/com/wire/kalium/persistence/Users.sq

```diff
@@ -46,36 +46,39 @@ email = excluded.email,
 phone = excluded.phone,
 accent_id = excluded.accent_id,
 team = excluded.team,
-connection_status = excluded.connection_status,
 preview_asset_id = excluded.preview_asset_id,
 complete_asset_id = excluded.complete_asset_id,
-user_type = excluded.user_type,
```

Comment:
It's only relevant to update this field for team user and then it should be done with `upsertTeamMemberUserType`

Reply:
Ok, turns out this is not only erlagen for team users and thus I've reverted this and now it's instead mandatory to provide a correct `userType` when upserting a user. We weren't providing a correct `userType` when syncing team users in the `UserRepository`, which I've fixed for now by fetching it from the local db.

Reply:
cc @vitorhugods @yamilmedina @mchenani